lang/cs/Org.Apache.REEF.Wake.Tests/StreamingTransportTest.cs

```diff
@@ -168,5 +189,16 @@ private static ITcpPortProvider GetTcpProvider(int portRangeStart, int portRange
                 .Build();
             return TangFactory.GetTang().NewInjector(configuration).GetInstance<ITcpPortProvider>();
         }
+
+        private static ITcpClientFactory GetTcpClientFactory(int connectionRetryCount, int sleepTimeInMs)
+        {
+            var config =
```

Comment:
Use TcpClientConfigurationModule to set configuration would be better as that is the way you would use in setting configurations for it.